src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -265,8 +265,8 @@ export class InstructorHomePageComponent extends InstructorSessionBasePageCompon
           strB = b.course.courseId;
           break;
         case SortBy.COURSE_CREATION_DATE:
-          strA = a.course.creationDate;
-          strB = b.course.creationDate;
+          strA = String(a.course.creationDate);
+          strB = String(b.course.creationDate);
```

Comment:
Perhaps you need to manually test that the date will not be displayed as a long number in all pages that use `Course` DTO, as I search and find toEpochMilli() will convert the time to a `number` instance

Reply:
Thanks for the fast review! I double checked with the pages and it does not affect as the `endpoint` to obtain `creationDate` is only used in `InstructorHome`